src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,70 @@
+package duke;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+import duke.command.Command;
+
+/**
+ * Duke is the main class that handles the overall execution of the program.
+ *
+ * @author Loh Wen Hao Aaron
+ *
+ */
+public class Duke extends Application {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui = new Ui(this);
+
+    /**
+     * Duke constructor that takes in the file path for the save file.
+     *
+     */
+    public Duke() {
+
+    }
+
+    private void initialiseDuke(String filePath) {
+        try {
+            this.storage = new Storage(filePath);
+            this.tasks = new TaskList(storage.produceReadableString(), this.storage, this.ui);
+        } catch (DukeException e) {
+            System.out.println("Save file could not be loaded!");
+            System.out.println("Generating empty task list!");
+            this.tasks = new TaskList(this.ui);
+        }
+    }
+
+    /**
+     * This method runs the program.
```

Comment:
Maybe this could just be changed to:
```suggestion
     * Runs the program.
```